producer/writer/options.go

```diff
@@ -32,15 +32,14 @@ import (
 )
 
 const (
-	defaultDialTimeout                = 10 * time.Second
-	defaultMessageRetryBackoff        = 5 * time.Second
-	defaultPlacementWatchInitTimeout  = 5 * time.Second
-	defaultTopicWatchInitTimeout      = 5 * time.Second
-	defaultCloseCheckInterval         = 2 * time.Second
-	defaultConnectionResetDelay       = 2 * time.Second
-	defaultMessageRetryWriteBatchSize = 64
+	defaultDialTimeout               = 10 * time.Second
+	defaultMessageRetryBackoff       = 5 * time.Second
+	defaultPlacementWatchInitTimeout = 5 * time.Second
+	defaultTopicWatchInitTimeout     = 5 * time.Second
+	defaultCloseCheckInterval        = 2 * time.Second
+	defaultConnectionResetDelay      = 2 * time.Second
 	// Iterate and remove 1M list item cost about 160ms.
-	defaultMessageRetryIterateBatchSize = 1024 * 1024
+	defaultMessageRetryBatchSize = 1024 * 1024
```

Comment:
This is a very large batch size, perhaps make it smaller by default?

Reply:
sounds good